js_modules/dagit/packages/core/src/instance/backfill/BackfillPage.tsx

```diff
@@ -184,31 +184,75 @@ export const BackfillPage = () => {
           </thead>
           <tbody>
             {backfill.assetBackfillData?.assetBackfillStatuses.map((asset) => {
-              return asset.__typename === 'AssetPartitionsStatusCounts' ? (
-                <tr key={asset.assetKey.path.join('/')}>
-                  <td>
-                    <Box flex={{direction: 'row', justifyContent: 'space-between'}}>
-                      <div>
-                        <a href={assetDetailsPathForKey(asset.assetKey)}>
-                          {asset.assetKey.path.join('/')}
-                        </a>
-                      </div>
-                      <div>
-                        <StatusBar
-                          targeted={asset.numPartitionsTargeted}
-                          requested={asset.numPartitionsInProgress}
-                          completed={asset.numPartitionsMaterialized}
-                          failed={asset.numPartitionsFailed}
-                        />
-                      </div>
-                    </Box>
-                  </td>
-                  <td>{asset.numPartitionsTargeted}</td>
-                  <td>{asset.numPartitionsInProgress}</td>
-                  <td>{asset.numPartitionsMaterialized}</td>
-                  <td>{asset.numPartitionsFailed}</td>
-                </tr>
-              ) : null;
+              if (
+                ['AssetPartitionsStatusCounts', 'UnpartitionedAssetStatus'].includes(
+                  asset.__typename,
+                )
+              ) {
+                let targeted;
+                let inProgress;
+                let completed;
+                let failed;
+                if (asset.__typename === 'AssetPartitionsStatusCounts') {
+                  targeted = asset.numPartitionsTargeted;
+                  inProgress = asset.numPartitionsInProgress;
+                  completed = asset.numPartitionsMaterialized;
+                  failed = asset.numPartitionsFailed;
+                } else {
+                  targeted = 1;
+                  failed = asset.failed ? 1 : 0;
+                  inProgress = asset.inProgress ? 1 : 0;
+                  completed = asset.materialized ? 1 : 0;
+                }
+                return (
+                  <tr
+                    key={asset.assetKey.path.join('/')}
+                    data-testid={testId(`backfill-asset-row-${asset.assetKey.path.join('/')}`)}
```

Comment:
It's not super important but it'd be nice to use `tokenForAssetKey(asset.assetKey)` here - it does the same thing but I've been trying to limit the surface area of the `/` in case this logic gets more complicated someday